### ISSUE TITLE
MD Render Pane does not display BMP files. RTE does.

### ISSUE THREAD

**Issue by dpoulton-github** [laurent22/joplin#10914]
### Operating system

Linux

### Joplin version

3.0.15

### Desktop version info

Joplin 3.0.15 (prod, linux)
Sync Version: 3
Profile Version: 47
Keychain Supported: No
Revision: 598677b

Backup: 1.4.1

OS specifics: Linux Mint 21.3 Cinnamon, Kubuntu 22.04, Windows.
Sync method / target: None

### Current behaviour

1. Import a BMP image into a Markdown note.
2. Joplin does not add "!" before the image link as it treats BMP as an image type it cannot display.
3. Without an "!" the image appears as expected in the render pane as a generic file icon followed by the file name. *(fig.1)*
4. Add "!" before the link. As expected the render pane shows a broken image icon is displayed followed by the file name. *(fig.1)*
5. Switch to the Rich Text Editor (RTE)
7. The image link prefixed with "!" **IS** displayed. *(fig.2)*
8. A BMP file imported directly into a Rich-Text note will not display, but that is only because the underlying Markdown link does not automatically have "!" added as a prefix. Adding one causes the image to display. 

*Fig.1 - Markdown Editor*
![image](https://github.com/user-attachments/assets/66b5d6b3-5f45-4b5b-8b3b-8ffccdd597ca)

*Fig.2 - Rich-Text Editor*
![image](https://github.com/user-attachments/assets/1b735729-0711-468d-865d-0d993430a614)

## Other Information

- Linked forum post > [Showing [Image: :/…] instead images itself in the preview](https://discourse.joplinapp.org/t/showing-image-instead-images-itself-in-the-preview/39743)

### Expected behaviour

Image rendering should be the same for both editor types.

### Logs

N/A

**Comment by Rolograaf** [comment]
Joplin also stopped showing local jpg's `![imagename](file:///F:/path/image.jpg)` problem remains unchanged with new release v3.1.4 on Win10

**Comment by dpoulton-github** [comment]
@Rolograaf 

I think what you are describing is a different issue. It looks like the above one was solved by adding a BMP mime type to the list of accepted images.

I did manage to duplicate what you are seeing. For me a file URI (`[image.jpg](file:///path/to/image.jpg)`) does work on Windows and Linux in v3.0.15 but no longer works for both OSs with v3.1.4. I did not test anything other than JPGs.

Probably deserves its own issue rather than this closed one.
